config/dispatch.yaml.ctmpl

```diff
@@ -0,0 +1,40 @@
+{{ $environment := env "ENVIRONMENT"}}
+
+dispatch:
+{{if eq $environment "dev"}}
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: angio
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: mbc
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: brain
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: osteo
```

Comment:
These doesn't look right... shouldn't they be `dev.ascproject.org`, `dev.braincancerproject.org`, etc? But I guess this won't hinder us from pushing to prod...

Reply:
No, these urls have all changed after chatting with ropes.

Reply:
I mean "these urls are right".  Sorry!